src/main/java/com/customatics/leaptest_plugin/PluginHandler.java

```diff
@@ -462,7 +462,7 @@ public void createJUnitReport(FilePath workspace, String JUnitReportFile, final
             if(workspace.isRemote())
             {
                 VirtualChannel channel = workspace.getChannel();
-                reportFile = new FilePath(channel, Paths.get(workspace.toURI().getPath(), JUnitReportFile).toString());
+                reportFile = new FilePath(channel, workspace.toURI().getPath() + "/" + JUnitReportFile);
```

Comment:
We will try it on Windows and Linux using `File.separator` instead of `"/".` If succeed, maybe, will be changed in the next update. But I still don't understand why this issue happens on Windows system considering documentation for `Paths.get `method:
1. https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html
2. http://www.java2s.com/Tutorials/Java/java.nio.file/Paths/Java_Paths_get_String_first_String_more_.htm